svgdigitizer/entrypoint.py

```diff
@@ -10,12 +10,15 @@
     Options:
       --help  Show this message and exit.
     Commands:
-      create-svg  Write an SVG that shows `png` or `jpeg` as a linked image.
-      cv          Digitize a cylic voltammogram and create a frictionless...
-      digitize    Digitize a 2D plot.
-      figure      Digitize a figure with units on the axis and create a...
-      paginate    Render PDF pages as individual SVG files with linked PNG images.
-      plot        Display a plot of the data traced in an SVG.
+      create-svg     Write an SVG that shows `png` or `jpeg` as a linked image.
+      cv             Digitize a cylic voltammogram and create a frictionless...
+      digitize       Digitize a 2D plot.
+      figure         Digitize a figure with units on the axis and create a...
+      get-citation   Get the citation from the DOI provided PDF file.
```

Comment:
I think we usually denote this as biblography.
```suggestion
      get-citation   Get the bibtex style bibliography entry from the DOI in the provided PDF file.
```